docs/how-to/contribute-docs.rst

```diff
@@ -20,18 +20,9 @@ the other index files for clues.
 Build documentation locally
 ---------------------------
 
-Go to the documentation folder::
+To build the documentation locally, run::
 
-    $ cd docs
+    $ tox -e docs
 
-Install Sphinx with the helpers and extensions we use::
-
-    $ pip install -r requirements_rtd.txt
-
-Create the static files: ::
-
-    $ make html
-
-Check for any errors and,if possible, fix them.
-The output of the file should be in the ``_build/html`` folder.
-Open the file you changed in the browser.
+The output of the file should be in the ``_build/html`` folder. If there are
```

Comment:
This is no longer correct when using `python setup.py build_sphinx` - the default build location in that case is build/html in the project root directory, rather than docs/_build/html.

Reply:
Looks like it's `build/sphinx/html` - but updated.